drivers/bluetooth/hci/Kconfig

```diff
@@ -134,3 +147,9 @@ config BT_HCI_SETUP
 	  The user should generally avoid changing it via menuconfig or in
 	  configuration files. This option are enabled by the vendor-specific
 	  HCI extension, where the Setup function is implemented.
+
+if BT_CYW43XXX
+
+source "drivers/bluetooth/hci/Kconfig.infineon"
+
+endif # BT_CYW43XXX
```

Comment:
I think it would be a better pattern to always include the leaf file (Kconfig.infineon), then having the main option (BT_CYW43XXX) as a menuconfig depending on BT_H4, and the other option under it. For example: https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/serial/Kconfig.nrfx. Main advantage is that then you have all the relevant option under the same file (which maybe should match the driver name and be called Kconfig.cyw43xxx)

Reply:
@jhedberg ^^

Reply:
done